test/IXUrlParserTest.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ *  IXSocketTest.cpp
+ *  Author: Benjamin Sergeant
```

Comment:
Don't you want to add your name in there for posterity ?

Reply:
Oki